infrastructure/zk/src/run.ts

```diff
@@ -28,6 +28,18 @@ export async function deployERC20AndWeth({
                 { "name": "DAI",  "symbol": "DAI",  "decimals": 18 },
                 { "name": "wBTC", "symbol": "wBTC", "decimals":  8, "implementation": "RevertTransferERC20" },
                 { "name": "BAT",  "symbol": "BAT",  "decimals": 18 },
+                { "name": "GNT",  "symbol": "GNT",  "decimals": 18 },
```

Comment:
I think these changes are here by accident, IIRC these lines were removed recently. Please check what's the relevant part of the diff here and make sure to remove everything else.

Same in other files.

Reply:
Changed for the tokens used in main.